test/stdlib/TestData.swift

```diff
@@ -3838,7 +3838,8 @@ class TestData : TestDataSuper {
     }
 
     func test_increaseCount() {
-        guard #available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *) else { return }
+        // https://github.com/apple/swift/pull/28919
```

Comment:
Is this comment needed?

Reply:
Going to remove this
```suggestion

```

Reply:
This pointed to the change that introduced the behavior the case is testing, explaining the unusual availability incantation.